src/syntax.ml

```diff
@@ -145,13 +145,14 @@ type p_cmd =
   (** Require statement. *)
   | P_open       of module_path
   (** Open statement. *)
-  | P_symbol     of symtag list * ident * p_type
+  | P_symbol     of symtag list * ident * p_arg list * p_type
   (** Symbol declaration. *)
   | P_rules      of p_rule list
   (** Rewriting rule declarations. *)
   | P_definition of bool * ident * p_arg list * p_type option * p_term
   (** Definition of a symbol (unfoldable). *)
-  | P_theorem    of (ident * p_type) loc * p_tactic list * p_proof_end loc
+  | P_theorem
+    of (ident * p_arg list * p_type) loc * p_tactic list * p_proof_end loc
```

Comment:
We should define a type `p_statement` for the first argument:
```ocaml
type p_statement = (ident * p_arg list * p_type) loc
```
And this way the declaration will fit again on one line.